README.md

```diff
@@ -1,4 +1,4 @@
-# Awesome Atom [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+# Awesome Atom [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) [![Javascript](https://badges.frapsoft.com/javascript/code/javascript.png?v=100)](https://github.com/ellerbrock/javascript-badges/)
```

Comment:
I thought looks cool next to the Awesome Badge because its a JavaScript Project. 
In case you don't like it you can delete it ...